service/worker/scanner/executions/activities.go

```diff
@@ -55,6 +55,11 @@ const (
 	ErrSerialization = "encountered serialization error"
 )
 
+var scanTypePrefixMap = map[common.ScanType]string{
+	common.ConcreteExecutionType: "", // leave it empty for now to be backwards compatible
```

Comment:
I don't think there should be any backwards compatible concerns. We can also just terminate the current run.

Reply:
Or update the dashboards.

Reply:
Lets do it safe and update it in "register the scanner workflow configured for current executions" step respectively.

Reply:
I think `scanTypePrefixMap` is only used inside activity implementation so there's no backward compatibility issue here? Can you elaborate a little bit on your concern here? 

The only backward compatibility issue I can think of is the newly added `scanType` field in WorkflowParams and ActivityParams. But it seems like you rely on the fact that `ConcreteExecutionType` is the default value for that field to resolve the issue?

Reply:
Its about the metric scope. Discussed offline.